src/components/RecipiesList.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Box, Image, Heading } from "@chakra-ui/react";
+import dish from "../dish.png";
```

Comment:
dar unde este acest dish.png?

Reply:
aceasta e o imagine, adaugata de mine in src

Reply:
am adaugat-o ca sa se afiseze in cazul cand info despre reteta vine fara imagine

Reply:
In general, acest api nu pre abunda in informatie, si nu prea ai ce lua, doar titlul imagine(care nu tot timpul exista), si link catre un website, descriere foarte putina